runtime/README.md

```diff
@@ -604,6 +604,29 @@ Table: `SHUTDOWN` output arguments
 | chksum       | u32       | Checksum over other output arguments, computed by Caliptra. Little endian.
 | fips_status  | u32       | Indicates if the command is FIPS approved or an error
 
+### ADD\_SUBJECT\_ALT\_NAME
```

Comment:
Would there be cases where the caller wants to set this extension for some keys but not others? e.g. SPDM responder v.s. CCA HES. I'm wondering if we should also have an "unset" mailbox command.

But also it's a non-critical extension so maybe it's not a big deal.

cc @varuns-nvidia

Reply:
For CSRs it's probably not a big deal since the CA can pick and choose the fields it cares about. For the leaf certificate though there may be Verifiers that care, and that might be strictly the presence or absence of the field.

Reply:
Agreed. It's probably best if we give the SoC a way to request this on specific calls.

One way we could do this is via an extra boolean `add_subject_alt_name` parameter to InvokeDpe. If true, use the name provided by `ADD_SUBJECT_ALT_NAME`. Otherwise don't use it. If the command is not CertifyKey, this flag is ignored.

To avoid changing the InvokeDpe command too much for stuff like this in the future, we could reserve a u32 of flags.

Reply:
Another idea: Could have a separate CERTIFY_KEY_EXTENDED command which takes these extra parameters and calls CertifyKey internally. This is an easier mechanism to get Caliptra to add extra informational things to the leaf cert.

Reply:
@steven-bellock @varuns-nvidia Any opinion on which route we should take here?

Reply:
I like Jordan's idea of CERTIFY_KEY_EXTENDED.

Reply:
Done